src/include/wire/libevent_thread.h

```diff
@@ -67,11 +71,11 @@ class LibeventThread {
 };
 
 class LibeventWorkerThread : public LibeventThread {
- private:
+ public:
   // New connection event
   struct event *new_conn_event_;
```

Comment:
Is it necessary to it public? If it has to be public, maybe you could change the name `new_conn_event_` to `new_conn_event`, because the naming conventions.